### PR TITLE
Add conda-forge channel to README/Docs

### PR DESCRIPTION
**Pull request by PetrovKP**
# Description
Please include a summary of the change. For large or complex changes please include enough information to introduce your change and explain motivation for it.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +67,17 @@ conda install scikit-learn-intelex -c intel
...
+
+#### 📦 Anaconda Cloud: Conda-Forge channel
+
+| OS / Python version     | **Python 3.6** | **Python 3.7** | **Python 3.8**| **Python 3.9**|
+| :-----------------------| :------------: | :------------: | :------------:| :------------:|
+|    **Linux**            |   [CPU]        |   [CPU]        |     [CPU]     |     [CPU]     |
+|    **Windows**          |   [CPU]        |   [CPU]        |     [CPU]     |     [CPU]     |
+|    **OsX**              |   [CPU]        |   [CPU]        |     [CPU]     |     [CPU]     |
 
 #### 📦 Anaconda Cloud: Intel channel
```
Comment (SmirnovEgorRu) [inline README.md:82]: In Intel channel we have python versions more than 3.7 (the same as PyPI)

File: INSTALL.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ cd <checkout-dir>
 python setup_sklearnex.py develop
 ```
 
-To install scikit-learn-intelex without the dependency on daal4py:
+To install scikit-learn-intelex without the download dependency of daal4py:
```
Comment (outoftardis) [inline INSTALL.md:25]:
```suggestion
To install scikit-learn-intelex without downloading daal4py:
```

File: doc/sources/get_started.rst [hunk 3]
```diff
@@ -125,12 +161,12 @@ To install the package in the development mode::
     cd <checkout-dir>
     python setup.py develop
 
-To install scikit-learn-intelex without the dependency on daal4py::
+To install scikit-learn-intelex without the download dependency of daal4py::
```
Comment (outoftardis) [inline doc/sources/get_started.rst:164]:
```suggestion
To install scikit-learn-intelex without downloading daal4py::
```

File: doc/sources/get_started.rst [hunk 4]
```diff
@@ -125,12 +161,12 @@ To install the package in the development mode::
...
 
     cd <checkout-dir>
     python setup_sklearnex.py install --single-version-externally-managed --record=record.txt
 
-To install scikit-learn-intelex without the dependency on daal4py in the development mode::
+To install scikit-learn-intelex without the download dependency of daal4py in the development mode::
```
Comment (outoftardis) [inline doc/sources/get_started.rst:169]:
```suggestion
To install scikit-learn-intelex without downloading daal4py in the development mode::
```

**Review by outoftardis — APPROVED** [review]
LGTM

**Pull request merged**